.gitignore

```diff
@@ -9,4 +9,6 @@ CMakeCache.txt
 CMakeFiles/
 Makefile
 cmake_install.cmake
-libmender-mcu-client.a
\ No newline at end of file
+libmender-mcu-client.a
+
+.DS_Store
```

Comment:
*noice* in the commit message. Interesting, I have never seen this file before. Should we add it everywhere across our projects? Or can you prevent if from being created?

Reply:
Apparently you can:
```
$ defaults write com.apple.desktopservices DSDontWriteNetworkStores true
```
This file specifies folder icon positions, and other visual information in the macOS Finder app (i.e., the file explorer). I'm guessing that most mac users would like to not disable it, although, I just did. It seems to be created automatically once you open a directory in the Finder app.

Reply:
It does not create noice in the commit message. But it gets listed as one of those untracked files when you do `git status`.